units/src/amount.rs

```diff
@@ -835,7 +835,7 @@ fn fmt_satoshi_in(
     Ok(())
 }
 
-/// Amount
+/// Amount.
```

Comment:
I think we shouldn't add periods to headings.

If you see technical docs
(for example, [RFC-6979: Deterministic Usage of the Digital Signature Algorithm (DSA) and Elliptic Curve Digital Signature Algorithm (ECDSA)](https://www.rfc-editor.org/rfc/rfc6979))
you'll see that the headings do not have periods.

Reply:
But they do "periodize" the item lists in several distinct forms:

- with the `;` followed by `.`  (2.3.1 first list)
- nothing (2.2)
- periods (several examples)

Reply:
I agree we shouldn't add periods to headings since this does seem to be the standard, and in most places this is what has been done.  The exception is in headings for modules and structs e.g. below where the headings are shown as the description in the list and are inconsistent:
![image](https://github.com/rust-bitcoin/rust-bitcoin/assets/108404580/b7b219e5-5a27-4597-9ad4-a39f077f1798)

These headings are also not marked as a heading with a `#` and appear small on the page e.g. "A Bitcoin address.":

![image](https://github.com/rust-bitcoin/rust-bitcoin/assets/108404580/d4f66118-e657-4694-97a4-9cbe430f18ba)


We could change them all with for example heading 2 `##` so it looks like below, note in the Struct list it is still displayed as standard text and not a heading style:
![image](https://github.com/rust-bitcoin/rust-bitcoin/assets/108404580/6f004edc-4cb9-4e04-991f-8c7311670ef5)


![image](https://github.com/rust-bitcoin/rust-bitcoin/assets/108404580/3c07c105-92a8-40f5-8116-58cfffaf8a4b)

Reply:
I see, so we should pick either one of the options and standardize everything.
I understand your argument and see several valid points.
I don't have a preference, @apoelstra and @tcharding any opinions?

Reply:
I took a quick glance at serde, which uses periods. We should also make sure we're using indefinite articles on each one (so "Amount" becomes "An amount."

We should not use `##` or any other heading. We had that a long time ago and somebody showed up here and lectured me about it, so I took them away. The idea is that these are not headings but a "summary description" of the types.